blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -801,26 +807,35 @@ def treat_bone(b, parent = None):
         for mesh in range (1, len(meshes)):
           for submesh in meshes[mesh].submeshes:
             submesh.bindtomesh(meshes[0])
+      checkPath(modelFilePath)#python does not create directory just the file. This function added to create t eh directory
       try:
         file = open(modelFilePath, 'w')
-      except IOError:
-        errmsg = "IOError " #%s: %s" % (errno, strerror)
+      except Exception as e:
+        print(str(e))
+        # errmsg = "IOError " #%s: %s" % (errno, strerror)

```

Comment:
Commented out code looks like debug code. Your change is an improvement as errmsg does not seem to get used at all so the exception basically just got swallowed.

To me it seems like it might be best to just not catch the exception.